articles/views.py

```diff
@@ -37,3 +40,11 @@ class ArticleViewSet(viewsets.ModelViewSet):
 
     permission_classes = [IsAdminUser]
     http_method_names = VALID_HTTP_METHODS
+
+    @method_decorator(
+        cache_page_for_all_users(
+            settings.SEARCH_PAGE_CACHE_DURATION, cache="redis", key_prefix="search"
+        )
+    )
+    def list(self, request, *args, **kwargs):
+        return super().list(request, *args, **kwargs)
```

Comment:
Probably want to clear this cache when a new article is  created, edited, or deleted

Create (no UI yet, just via api): http://api.open.odl.local:8063/api/v1/articles/

Editing UI (after created above): http://open.odl.local:8062/articles/<aricled_id>/edit